WalletWasabi/Helpers/Secp256k1Helpers.cs

```diff
@@ -0,0 +1,13 @@
+using NBitcoin.Secp256k1;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WalletWasabi.Helpers
+{
+	public static class Secp256k1Helpers
+	{
+		public static bool Equals(GE a, GE b)
+			=> a.IsInfinity == b.IsInfinity && (a.x == b.x && a.y == b.y);
+	}
```

Comment:
If GE uses affine coordinates then why do you need to check whether they're the neutral element?

Reply:
Never look at a draft PR. It's a proof of concept draft, the `WabiSabi` branch is way ahead of this. The equality things are already sorted out with the `GroupElement` encapsulation and you were reviewing it, too.